File: src/kyu_6/two_sum/csharp/solution.cs
public class Kata
{
  public static int[] TwoSum(int[] numbers, int target)
  {
    for(int i = 0; i < numbers.Length - 1; i++)
    {
      for(int u = i + 1; u < numbers.Length; u++)
      {
        if(numbers[i] + numbers[u] == target)
          return new int[]{ i, u };
      }
    }
    return null;
  }
}

File: src/kyu_6/two_sum/java/Solution.java
package kyu_6.two_sum.java;

public class Solution {
    public static int[] twoSum(int[] numbers, int target) {
        for (int i = 0; i < numbers.length; i++) {
            for (int j = 0; j < numbers.length; j++) {
                if (i != j) {
                    if (numbers[i] + numbers[j] == target) {
                        return new int[]{i, j};
                    }
                }
            }
        }
        return new int[]{0, 0};
    }
}
